Plugins/FancyMesh/src/FancyMeshComponent.cpp

```diff
@@ -211,6 +218,11 @@ namespace FancyMeshPlugin
         return &(getMesh());
     }
 
+    const std::vector<uint>* FancyMeshComponent::getDuplicateTableOutput() const
```

Comment:
I don't like the idea of returning a pointer like that (allows calls to delete). What about returning a const reference ?

Reply:
This has to be so because of the way the ComponentMessenger works.
Better ask @louen about that